### DISCUSSION TITLE
Dockerless installation

### DISCUSSION THREAD

**Discussion by markc** [NginxProxyManager/nginx-proxy-manager#2219] [Ideas]
I have no intention of ever using docker, as I have Proxmox and LXD containers and VMs out the wazoo. Is it possible to provide instructions or a brief guide on how to set up this proxy manager on a standalone server (already running nginx) without requiring docker, please?

**Comment by johntdavis84** [comment]
Not exactly what you had in mind, but have you considered just running a small LXD container with the bare minimum linux configuration to support docker, running NGINX Proxy Manager?

I cannot speak for the developer, but my recollection last time this came up was that NGINX Proxy Manager without Docker is an unsupported configuration.

-- -- --
John T Davis
***@***.***

**Comment by parisjp** [comment]
Try this: [https://github.com/ej52/proxmox-scripts/blob/main/lxc/nginx-proxy-manager/README.md](https://github.com/ej52/proxmox-scripts/blob/main/lxc/nginx-proxy-manager/README.md)

Worked for me, but not the script, see at the end the manual install

**Comment by Hotte512** [comment]
Yeah 👍 I dislike docker

**Comment by roni-estein** [comment]
This is a bit old, but are there non-docker installation instructions? I had a series of 7 apps running in docker containers about 4 years ago on my pi4 and it was overloaded. I have since reinstalled dietpi and the same systems barely break break 10% of system resources. Having 10 docker containers filled with tiny systems is still heavy for the pi. It would be great if there were just some trustable bare metal installation instructions.

Thanks

**Comment by DRF-Solutions** [comment]
Yes, I too would appreciate a non-Docker based installation script. I always end up having to completely overhaul anything docker based as docker can be a PITA to keep happy. Anything that requires accurate date & time you can basically forget using in docker... I'm on a mission in my lab to rid it of anything bound by docker and snaps (Ubuntu) as they're harder to maintain, harder to update and don't need to be in docker!

**Comment by brian78675** [comment]
Docker does not work for my install as I cant connect to external systems without troublesome network definitions. This would be great

**Reply by khnjrdm** [reply]
This might be of interest for you then: https://github.com/njordium/npm-native
